### ISSUE TITLE
Inactive state clarification

### ISSUE THREAD

**Issue by Ethan1983** [Khang-NT/Kotlin-Coroutines-Lifecycle-Aware#4]
Thanks for this one when i was about to write one myself :-). Anyways, please clarify this in README.

>> This channel won't notify new value when lifecycleOwner is inactive (e.g Activity/Fragment paused)

LiveData does notify on a paused state. This is necessary when one activity is partially seen below another activity and in that case bottom activity will be paused but not stoped and visible part of the paused activity can still show UI updates.

**Comment by Khang-NT** [comment]
Hi, `master` branch is out of date. Please see `develop` branch instead.
https://github.com/Khang-NT/Kotlin-Coroutines-Lifecycle-Aware/tree/develop

With current implementation, `active` state is when `LifecycleOwner#currentState >= Lifecycle.State.STARTED`
```kotlin
private val isAtLeastStarted: Boolean
        get() = currentState.isAtLeast(Lifecycle.State.STARTED)
```

This behavior is same as document of `LiveData`
> LifecycleOwner is considered as active, if its state is STARTED or RESUMED.

So you can't  receive data after `onPause`, it's weird if `LiveData` does notify after `onPause`. 

Your use case is common, but it's an exception of this behavior, then I think you have to control lifecycle manually.

**Comment by Ethan1983** [comment]
Yeah, it claims either or and not both (started and resumed) for it to be active.

Anyways, try this app. It has a second activity launched as a dialog and first activity is notified (you can check the logs for the same)

https://github.com/Ethan1983/LiveDataStoppedActivity

**Comment by Khang-NT** [comment]
Yeah, you're right.
I just realized that when the `AppCompatActivity` partial display, `onPause` is invoked but `currentState` value still equal `STARTED`.

We treat the document so hard, but we forget `LifecycleOwner` is just an abstract class, the implementation maybe different a bit as it documented, sound like a bug but actually it's a feature :D.

Then both my code and `LiveData` will emit value as long as lifecycle state is `STARTED` or `RESUMED`, it covers your use case since the activity still in `STARTED` state when it partial display.

**Comment by Ethan1983** [comment]
Sounds good. Please consider updating your readme (stopped instead of paused), it will make it clear.

"This channel won't notify new value when lifecycleOwner is inactive (e.g Activity/Fragment paused)"
